msvc/vsimporter-templates/WinStore10-App/App.xaml.cpp

```diff
@@ -61,8 +57,7 @@ void App::OnActivated(Windows::ApplicationModel::Activation::IActivatedEventArgs
 /// </summary>
 /// <param name="sender">The source of the suspend request.</param>
 /// <param name="e">Details about the suspend request.</param>
-void App::OnSuspending(Object^ /*sender*/, SuspendingEventArgs^ /*e*/)
-{
+void App::OnSuspending(Object^ /*sender*/, SuspendingEventArgs^ /*e*/) {
     // TODO: Save application state and stop any background activity
```

Comment:
The way we add our events means we don't need to call them directly


---
In reply to: [72310498](https://github.com/Microsoft/WinObjC/pull/686#discussion_r72310498) [](ancestors = 72310498)